### ISSUE TITLE
[MobAmputation] Suggestion: Add compatibility with Flan's mod bullets?

### ISSUE THREAD

**Issue by PsychoLynx** [iChun/IssueTracker-Mods#10]
Hello! I'm a fan of both MobAmputation and Flan's mod (http://flansmod.com/), the latter of which adds numerous firearms to Minecraft. However, I find it disappointing that I can't seem to be able to add the bullet entities in the list of projectiles that affect gibbing in the .cfg. I'm using version 4.0.0 of MobAmputation and 4.8.0 of Flan's mod.

I asked about this problem in the Flan's mod modelling community forums: http://www.minecraft-smp.de/e107_plugins/forum/forum_viewtopic.php?46692.last 

Basically I got told to make a hook with Flan's mod, but being a Java noob, I have no idea how to do that. Anyway, here's also the source for Flan's mod: https://github.com/FlansMods/FlansMod

I hope I've added enough information here to be helpful! Thanks for reading!

**Comment by iChun** [comment]
In the MC 1.8 version of this mod, the projectile config has been changed to allow Class names as well as Entity names. There will also be a new config that will allow global gibbing from the use of projectiles, although I've not tested both of these. It may not work for different reasons but this is the best I can do, sorry :/

**Comment by PsychoLynx** [comment]
Hey, no worries! Thank you so much for the help :)
